interceptors/Mementifier.cfc

```diff
@@ -264,6 +266,8 @@ component {
 				arguments.item
 			);
 		} );
+		arguments.includes = listToArray( arrayToList( createObject("java", "java.util.HashSet").init( arguments.includes ).toArray() ) );
```

Comment:
Since you are using a `for` loop, in lieu of a member function to loop these objects, there should be no need for the  `listToArray( arrayToList(... ) )` part of these two lines.

Reply:
Mementifier's `$buildNestedMementoList()` method needs a native Lucee array or else its `.filter()` and `.map()` member will error out. Should I rewrite those, or leave this as-is?

Reply:
Probably have to leave them as is, or you could possibly use  `[].append( createObject("java", "java.util.HashSet").init( arguments.includes ).toArray(), true )`

Reply:
Then you wouldn't have to worry about any commas in items in the one-dimensional array

Reply:
Oh, very nice. Yeah, I'll test that out and commit / push up.

Reply:
Yeah, @jclausen `arrayAppend` / `[].append()` modifies the original array and returns a boolean, not the new array. I can't think of any (other) simple one-liner to convert to an array, so I'm going to leave this as-is for now.